Configuration/TCA/tx_t3monitoring_domain_model_client.php

```diff
@@ -90,6 +91,40 @@
                 'eval' => 'trim,password'
             ],
         ],
+        'host_header' => [
+            'exclude' => true,
```

Comment:
I guess we could remove all exclude stuff, makes no sense

Reply:
I would leave it. We have non-admin users. or do you mean this is not evaluated in the module of the extension?